src/test/run-pass/allocator-default.rs

```diff
@@ -10,7 +10,7 @@
 
 #![feature(alloc_jemalloc)]
 
-#[cfg(any(target_os = "linux", target_os = "macos"))]
+#[cfg(any(all(target_os = "linux", not(any(target_arch = "aarch64", target_arch = "mips", target_arch = "mips64", target_arch = "powerpc", target_arch = "powerpc64", target_arch = "s390x"))), target_os = "macos"))]
```

Comment:
In this particular case, we're simply disabling jemalloc-related tests that are not applicable on those platforms, because rust doesn't use jemalloc there. But in general, I do hope rust starts running tests on [these other platforms](https://forge.rust-lang.org/platform-support.html) too.